downloader/src/funTest/kotlin/SubversionDownloadTest.kt

```diff
@@ -93,6 +95,24 @@ class SubversionDownloadTest : StringSpec() {
             actualFiles.joinToString("\n") shouldBe expectedFiles.joinToString("\n")
         }.config(enabled = Subversion.isInPath(), tags = setOf(ExpensiveTag))
 
+        "Subversion can download a given tag" {
```

Comment:
The name is confusing, actually this is testing if "Subversion can download a path that points to a tag". Also we should have the same test where "SendMessage-1.0.2" is provided as `revision` because this should also work, right?

Reply:
My feeling is that "Subversion can download a path that points to a tag" also doesn't quite hit it. It exposes an implementation detail that in Subversion tags are paths, and currently test titles are designed to be more or less the same across VCS implementations.

> Also we should have the same test where "SendMessage-1.0.2" is provided as revision because this should also work, right?

No, that's exactly the interesting bit. The `svn` client does not support passing tag names as revisions via `-r`. If you want to checkout a tag, you omit `-r` and just checkout the tag path, not knowing which revision you would be getting.

I had considered to still support passing a tag name as the revision on our level as a convenience, but decided that this would be too much magic.

Reply:
Interesting, I agree about the magic.